apps/client/src/app/pages/resources/overview/resources-overview.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <h1 class="h3 mb-4 text-center" i18n>Resources Overview</h1>
```

Comment:
Resources (remove Overview)

Reply:
So should I render one of the markets, Glossary or Guides page on initial render?

Reply:
No, just adapt the text of `h1` to "Resources ~~Overview~~"